tests/integration/security/filebased_tls_origination/main_test.go

```diff
@@ -59,14 +57,17 @@ func setupConfig(cfg *istio.Config) {
 components:
   egressGateways:
   - enabled: true
-  ingressGateways:
```

Comment:
Any particular reason why we want to enable ingress gw for this test suite? IIRC it's not used in this test plus waiting for it to be ready would just add a few seconds to a already very long security test. I would suggest we do this for remote clusters as well if not needed

Reply:
I will change things to conditionally enable igw in multicluster only; for shared control plane in multicluster we use the ingress gateway for discovery.